cfripper/cli.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import sys
+from io import TextIOWrapper
+from itertools import chain
+from pathlib import Path
+from typing import Dict, Optional
+
+import click
+import pycfmodel
+from pycfmodel.model.cf_model import CFModel
+
+from cfripper.__version__ import __version__
+from cfripper.config.config import Config
+from cfripper.model.result import Result
+from cfripper.model.utils import convert_json_or_yaml_to_dict
+from cfripper.rule_processor import RuleProcessor
+from cfripper.rules import DEFAULT_RULES
+
+LOGGING_LEVELS = {
+    "ERROR": logging.ERROR,
+    "WARNING": logging.WARNING,
+    "INFO": logging.INFO,
+    "DEBUG": logging.DEBUG,
+}
+
+
+def setup_logging(level: str) -> None:
+    logging.basicConfig(level=LOGGING_LEVELS[level], format="%(message)s")
+
+
+def get_cfmodel(template: TextIOWrapper) -> CFModel:
+    template = convert_json_or_yaml_to_dict(template.read())
+    cfmodel = pycfmodel.parse(template)
+    return cfmodel
+
+
+def process_template(
```

Comment:
Split this into smaller chunks:
* Init
* Process
* Format result
* Output handling